lib/firehose/long_poll_transport.coffee

```diff
@@ -108,7 +108,7 @@ class LongPollTransport extends Transport
 
     unless @_needToNotifyOfReconnect or @_stopRequestLoop
       @_needToNotifyOfReconnect = true
-      @config.disconnected()
+      @config.disconnected(jqXhr, status, error)
```

Comment:
What type is the `event` that's passed to this from the websocket versions? As I'd at least like the first parameter to be a similar type across them both, my guess that in this case `error` may match better as a first argument.